index.php

```diff
@@ -0,0 +1,2 @@
+<?php
+echo "hello git";
```

Comment:
Please don't use double quotes for a simple string. Double quotes uses to process variables inside the string in php for example. It is redundant here.

Reply:
ok i understand, aand remember about it.I was a little bit confused during our call